cinema/serializers.py

```diff
@@ -59,6 +68,7 @@ class MovieSessionListSerializer(MovieSessionSerializer):
     cinema_hall_capacity = serializers.IntegerField(
         source="cinema_hall.capacity", read_only=True
     )
+    tickets_available = serializers.IntegerField(read_only=True)
```

Comment:
Ensure that the `tickets_available` field is correctly calculated and populated. This field should reflect the number of available tickets for a movie session, which might require additional logic to compute based on the total capacity and the number of tickets already sold.